source/slang-capture-replay/replay/decoder-consumer.h

```diff
@@ -0,0 +1,178 @@
+#ifndef DECODER_CONSUMER_H
+#define DECODER_CONSUMER_H
+
+#include "../../../slang.h"
+// #include "../../slang/slang-compiler.h"
+#include "../util/capture-format.h"
+
+namespace SlangCapture
+{
+    class IDecoderConsumer
+    {
+    public:
+        virtual void ICreateGlobalSession(ObjectID objectId, const uint8_t* parameterBuffer, int64_t bufferSize) = 0;
```

Comment:
This is a weird name. Should be `createGlobalSession`.

Reply:
act